### PR TITLE
🎉  add supremo cwl

### PR DESCRIPTION
**Pull request by dmiller15**
## Description

Adds the supremo cwl tool to the repo.

Part of https://github.com/d3b-center/bixu-tracker/issues/1692

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] It has a production run already: https://cavatica.sbgenomics.com/u/d3b-bixu-ops/d3b-scoring-cbtn-svs-with-supremo-research-project/tasks/789f6075-3642-4ba1-b609-cca037eff8b3/

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [x] I have committed any related changes to the PR

### CODE REVIEW

File: tools/supremo.cwl [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  and for scoring variants for disruption to genome folding.
+requirements:
+  - class: InlineJavascriptRequirement
+  - class: ShellCommandRequirement
+  - class: ResourceRequirement
+    ramMin: $(inputs.ram*1000)
+    coresMin: $(inputs.cpu)
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/supremo:1.0.0'
+baseCommand: [">&2", "python", "/opt/SuPreMo/scripts/SuPreMo.py"]
```
Comment (migbro) [inline tools/supremo.cwl:15]: Ah, you can lead with `>&2`? I forgot about that

**Review by migbro — APPROVED** [review]
Looks good! We can decide to tweak later - like make output a tar ball if we want.

File: tools/supremo.cwl (hunk 1 above)
Comment (dmiller15) [reply tools/supremo.cwl:15]: Yep. Helps clear things up sometimes.

**Comment by dmiller15** [comment]
@migbro True. I still don't know exactly what we should keep. Definitely the scores file. Probably the log file. One thing I'd like is if this tool actually annotated the VCF but 🤷

**Review by sickler-alex — APPROVED** [review]

**Pull request merged**